app/views/users/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email:") %>
+  <%= text_field_tag(:email) %>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %>
+
+  <%= submit_tag("Save") %>
```

Comment:
- Remember that you should be using `form_for` instead of `form_tag`. Comment out the `form_tag`.
See [(screenshot from Odin)](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_for.png)